.ci/run_tests.sh

```diff
@@ -7,11 +8,11 @@ createuser --createdb kong
 createdb -U kong kong_tests
 
 if [ "$TEST_SUITE" == "lint" ]; then
-    make lint
+    make lint &>> build.log || (cat build.log && exit 1)
 elif [ "$TEST_SUITE" == "unit" ]; then
-    make test
+    make test &>> build.log || (cat build.log && exit 1)
 elif [ "$TEST_SUITE" == "integration" ]; then
-    make test-integration
+    make test-integration &>> build.log || (cat build.log && exit 1)
 elif [ "$TEST_SUITE" == "plugins" ]; then
-    make test-plugins
+    make test-plugins &>> build.log || (cat build.log && exit 1)
```

Comment:
I would say let's leave the test output for now. The "noise" in this context is from deps, not tests per se.